engine/flink/avro-util/src/test/scala/pl/touk/nussknacker/engine/avro/KafkaAvroTestProcessConfigCreator.scala

```diff
@@ -24,12 +24,12 @@ object KafkaAvroTestProcessConfigCreator {
 
 class KafkaAvroTestProcessConfigCreator extends EmptyProcessConfigCreator {
 
-  protected val schemaRegistryProvider: SchemaRegistryProvider = createSchemaRegistryProvider
+  protected val testSchemaRegistryProvider: SchemaRegistryProvider = createSchemaRegistryProvider
```

Comment:
Is this rename was really needed? we are alread in `KafkaAvroTESTProcessConfigCreator`

Reply:
Change reverted. I had some confusion between `schemaRegistryProvider` in `KafkaAvroTestProcessConfigCreator` and `KafkaAvroSourceMixin`.